sdcm/utils/sstable/load_utils.py

```diff
@@ -228,3 +244,42 @@ def get_load_test_data_inventory(cls, column_number: int, big_sstable: bool,
             return MULTI_COLUMNS_DATA
 
         return []
+
+    @classmethod
+    def create_keyspace(cls, node, keyspace_name: str = "keyspace1", strategy: str = 'SimpleStrategy',  # pylint: disable=too-many-arguments
+                        replication_factor: int = 1, drop_ks_if_exists: bool = False):
+        if drop_ks_if_exists:
+            node.run_cqlsh(f"DROP KEYSPACE {keyspace_name}")
+
+        node.run_cqlsh("""
+                CREATE KEYSPACE %s WITH replication = {'class': '%s',
+                'replication_factor': %s}
+                """ % (keyspace_name, strategy, replication_factor))
+
+    @classmethod
+    def create_table_for_load(cls, node, schema_file_and_path: str):
+        schema = node.remoter.run(f"cat {schema_file_and_path}").stdout
+        node.run_cqlsh(schema.replace("\n", ""))
+
+    @classmethod
+    def validate_data_count_after_upload(cls, node, keyspace_name: str = "keyspace1", table_name: str = 'standard1'):
+        result = node.run_cqlsh(f"consistency QUORUM;SELECT COUNT(*) FROM {keyspace_name}.{table_name}")
+        return result.stdout
+
+
+class RemoteTemporaryFolder:
```

Comment:
This context manager is not tied to the sstables. So, it must be defined in another module with common utils.